.github/taskdef/prod-zkevm-mainnet.yaml

```diff
@@ -3,7 +3,7 @@ hostport: 80
 containerport: 80
 role: backend-apps
 environment: prod
-iac: terraform-workspace-aws-prod-apps-eu-west-1-apps-auto-claim-polygon-technology
+iac: terraform-workspace-aws-prod-apps-eu-west-1-apps-zkevm-mainnet-auto-claim-polygon-technology
```

Comment:
Should be terraform-workspace-aws-prod-applications-eu-west-1-apps-auto-claim-zkevm-mainnet-auto-claim